### ISSUE TITLE
[ACQ] BOTA-ACQ : IL N'EST PLUS POSSIBLE D'AJOUTER UNE NOUVELLE ACQUISITION (plus de 10 000) CETTE ANNEE

### ISSUE THREAD

**Issue by cibotalista** [Computaceae-IT/cit-ticketing#886]
Url : https://jbp.botalista.community/#/acquisition/edit/
Username : paris

Description : Bonjour les DEV, 

Nous ne pouvons plus ajouter de nouvelles acquisition cette année (plus de 10 000 !!!). Je vous joindrai la capture d'écran via GItHub.  

Cordialement 

Vincent Lipa

**Comment by SiropOps** [comment]
Effectivement, il y a un problème avec les années 2017 et 2018 sûrement dû à la migration; il faudrait reprendre toutes les numéros d’acquisition / spécimen de ses années là ? ou on laisse et si vous avez des acquisitions de ces années là, vous post / anti daté ?
